extension/app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -19,12 +20,18 @@ type Subscription = {
 	rpcSocket: WebSocket
 }
 
+export type EthereumUnsignedTransactionWithWebsite = EthereumUnsignedTransaction & { website: Website }
```

Comment:
I am not a huge fan of this.  I would rather see an envelope type that contains the website and the transaction as separate parts:
```ts
export type EnvelopeNameTbd = { transaction: EthereumUnsignedTransaction, website: Website }
```